src/metaswitch/ellis/prov_tools/update_user.py

```diff
@@ -48,14 +48,22 @@ def main():
     parser.add_argument("-q", "--quiet", action="store_true", dest="quiet", help="don't display the user")
     parser.add_argument("--hsprov", metavar="IP:PORT", action="store", help="IP address and port of homestead-prov")
     parser.add_argument("--plaintext", action="store_true", help="store password in plaintext")
+    parser.add_argument("--ifc", metavar="iFC-FILE", action="store", dest="ifc_file", help="XML file containing the iFC")
+    parser.add_argument("--prefix", action="store", default="123", dest="twin_prefix", help="twin-prefix (default: 123)")
```

Comment:
create_user has already set a precedent.  I'd rather keep the option name changes restricted to update_user to minimise confusion for existing users.  Let me know if you think thats the wrong call